ansible/roles/ocp4-workload-migration/README.adoc

```diff
@@ -34,21 +34,10 @@ This workload does not require any variables. Following variables can be overrid
 |===
 | Variable | Default Value | Description
```

Comment:
Should `mig_migration_namespace` be mentioned in this list of variables?

Reply:
I think we want to leave it off for reasons mentioned in comment above, unsafe for user to modify at present but var may be useful later.